docs/src/man/split_apply_combine.md

```diff
@@ -133,28 +133,6 @@ julia> by(iris, :Species) do df
 │ 3   │ Iris-virginica  │ 5.552   │ 0.304588  │
 ```
 
-A second approach to the Split-Apply-Combine strategy is implemented in the `aggregate` function, which also takes three arguments: (1) a DataFrame, (2) one or more columns to split the DataFrame on, and (3) one or more functions that are used to compute a summary of each subset of the DataFrame. Each function is applied to each column that was not used to split the DataFrame, creating new columns of the form `$name_$function` like with `by` (see above). We show several examples of the `aggregate` function applied to the `iris` dataset below:
```

Comment:
Maybe show how to apply a function to each non-grouping column using `by`?

Reply:
added